Source/Core/VideoCommon/HiresTextures.h

```diff
@@ -17,7 +18,7 @@ enum class TextureFormat;
 std::set<std::string> GetTextureDirectoriesWithGameId(const std::string& root_directory,
                                                       const std::string& game_id);
 
-class HiresTexture
+class HiresTexture final : public CustomTextureData
```

Comment:
Can you instead of the inheritance use a CustomTextureData member in this class?